src/lite-yt-embed.js

```diff
@@ -102,7 +102,7 @@ class LiteYTEmbed extends HTMLElement {
         const iframeHTML = `
 <iframe width="560" height="315" frameborder="0"
   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen
-  src="https://www.youtube.com/embed/${this.videoId}?autoplay=1"
+  src="https://www.youtube.com/embed/${this.videoId}?autoplay=1&start=${this.start}"
```

Comment:
So... thought about this fresh this morning... We do need to wait on getting this support until we have better attribute observation support and dynamic query param generation. With the current flow and this being "always added" it is easy to get `"null", "undefined", or "NaN"` added into this parameter. While I *believe* YouTube would be guarded enough to just play from the beginning in that case, I still would err on the side of not sending known broken data around.